### ISSUE TITLE
UI elements lose transparency with BetterHUD

### ISSUE THREAD

**Issue by SnowyJune678** [FlansMods/FlansMod#1226]
I have created an identical issue report in the Better HUD repo: https://github.com/mccreery/better-hud/issues/89

**Describe the bug**
Using Flans mod and BetterHUD, certain modded (and vanilla) UI elements become opaque, even on the parts that are supposed to be transparent.

**To Reproduce**
Steps to reproduce the behaviour:
1. Install Flans Mod and BetterHUD together
2. See that certain UI elements (from either BetterHUD or vanilla) will lose its transparencies unless holding multiple of the same item in the offhand slot. Examples include the Naseua effect, the hunger bar, and the armor bar. certain BetterHUD elements such as Vignette and water drops also will lose any transparency. 

**Expected behaviour**
UI elements will not ose transparency when the two mods are used togather

**Screenshots**

![image](https://user-images.githubusercontent.com/23426461/81968689-bcc2f280-95ea-11ea-9de6-8b716446f898.png)
(yes, this image does have other mods, but I did narrow the issue down to BetterHUD and Flans)

**Versions (please complete the following information)**
 - Forge: 14.23.5.1854
 - Flan's Mod: 5.5.2
 - Better HUD 1.4.1

**Additional context**

Log: [gist](https://gist.github.com/0something0/b76837cf584027a435cbdc79efc5bcd0)
BetterHUD repo also has a similar bug report: https://github.com/mccreery/better-hud/issues/60

**Comment by mccreery** [comment]
Hi, Better HUD dev here. I have done some debugging and it looks like Flan's mod corrupts `GlStateManager` so that calls like `GlStateManager.enableBlend` have no effect because the manager thinks it's already enabled.

I did a quick check from the root of the zip download for this repo:

    $ grep -rPo 'GL11.(glDisable|glEnable)' . | wc -l
    52

There are 52 cases of OpenGL state (I only checked flags) being modified directly instead of using the manager. This is probably the cause of the issue.
